src/Helpers.py

```diff
@@ -26,3 +36,45 @@ def is_category_enabled(world: MultiWorld, player: int, category_name: str) -> b
             if not is_option_enabled(world, player, option_name):
                 return False
     return True
+
+def is_item_name_enabled(world: MultiWorld, player: int, item_name: str) -> bool:

```

Comment:
the location/item_name version wree made to work like is_category_enabled but still call the hook of the base not-name version, if its not something you like they can be removed and the rest will work.